Formula/binutils.rb

```diff
@@ -12,6 +13,11 @@ class Binutils < Formula
     sha256 "7c693e68ebfcd654fd781480e68469e8f5af93c7397754a8caae1937322feb8f" => :el_capitan
   end
 
+  # See https://sourceware.org/bugzilla/show_bug.cgi?id=23728 and
+  # https://github.com/Homebrew/homebrew-core/issues/32516
+  keg_only :provided_by_macos,
+           "because Apple provides binutils and gnm is broken and breaks gcc"
+
   # No --default-names option as it interferes with Homebrew builds.
```

Comment:
- Remove the comment, which is now irrelevant.
- The `--program-prefix=g` configure option should go too, since we don't put these tools in PATH anymore.

Reply:
@fxcoudert Which comment? The `# No --default-names ... ` one? Also, should we add back `--default-names`?

Reply:
Better to create a directory `libexec/bin` of symlinks with the default names, that users can add to their path if they desire.

Reply:
Also, I searched for explicit dependencies on binutils, and came up with 4 formula:

- [`crosstool-ng`]
- [`lightning`]
- [`pkgdiff`]
- [`pwntools`]

(I used `brew uses --include-build --include-test --include-optional --recursive binutils` to search.)

[`crosstool-ng`]: https://github.com/Homebrew/homebrew-core/blob/master/Formula/crosstool-ng
[`lightning`]: https://github.com/Homebrew/homebrew-core/blob/master/Formula/lightning
[`pkgdiff`]: https://github.com/Homebrew/homebrew-core/blob/master/Formula/pkgdiff
[`pwntools`]: https://github.com/Homebrew/homebrew-core/blob/master/Formula/pwntools.rb

Reply:
I went through the four formula above and none had any obvious adjustments that were required in the formula itself, however, virtually all of them require access to binutils binaries at runtime. Will the cause issues because the `binutils` executables are not on the default `PATH` anymore? Would formula revision bumps solve this, if it is indeed an issue?